Source/Analytics/Read/CaseReports/CaseReportsPerRegionLast4WeeksQuery.cs

```diff
@@ -5,16 +5,16 @@
 
 namespace Read.CaseReports
 {
-    public class CaseReportsPerRegionLast7DaysQuery : IQueryFor<CaseReportsPerRegionLast7Days>
+    public class CaseReportsPerRegionLast7DaysQuery : IQueryFor<CaseReportsPerRegionLast4Weeks>
```

Comment:
The name of the query needs to be renamed as well (i tried but couldn't find the documentation for what the dolittle command is aagain..).

Reply:
Aha, yes, this must be updated. I think it's the command `dolittle add query CaseReportsPerRegionLast4Weeks` – let that replace this one. The artifact ID will be updated, in the case that someone is using that manually – for everything else it should be updated. This one also needs to be removed from `artifacts.json`.